pkg/ibmcloudprovider/volume_provider.go

```diff
@@ -63,15 +64,18 @@ func NewIBMCloudStorageProvider(clusterVolumeLabel string, k8sClient *k8s_utils.
 
 	var clusterInfo utilsConfig.ClusterConfig
 	logger.Info("Fetching clusterInfo")
```

Comment:
Please move this log inside the if condition

Reply:
ok